addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -98,6 +98,34 @@ def test_event_booth_prices_with_sale_order(self):
                 booth.state, 'unavailable',
                 "Booth should not be available anymore.")
 
+    @users('user_sales_salesman')
+    def test_event_booth_count(self):
```

Comment:
Would rename this test to "test_event_booth_registrations_inverse" and test a bit more than just the count.